cogs/tibia.py

```diff
@@ -65,9 +65,14 @@ def __init__(self, bot: NabBot):
         embed = self.get_user_embed(ctx, user)
 
         # No user or char with that name
-        if char == ERROR_DOESNTEXIST and user is None:
-            await ctx.send("I don't see any user or character with that name.")
-            return
+        if char == ERROR_DOESNTEXIST:
```

Comment:
This would make it not look for an user's characters if there isn't a character with that same name. For instance, let's say my discord username is "Exiva is life" and I have several characters registered, if I used `/whois exiva is life`, since there's no character with that name, it would go into that if, and say it doesn't know who I am.